languages/csharp/exercises/concept/switch-statements/.docs/instructions.md

```diff
@@ -26,31 +26,31 @@ PlayAnalyzer.AnalyzeOnField(10);
 
 ### 2. Raise an alert if an unknown shirt number is encountered.
 
-Modify the `PlayAnalyzer.OffField()` method to throw an `ArgumentException` when a shirt number outside the range 1-11 is processed.
+Modify the `PlayAnalyzer.AnalyzeOnField()` method to throw an `ArgumentException` when a shirt number outside the range 1-11 is processed.
 
 ### 3. Extend the coverage to include off field activity
 
-Implement the `PlayAnalyzer.OffField()` method to output description of activities and characters around the field of play.
+Implement the `PlayAnalyzer.AnalyzeOffField()` method to output description of activities and characters around the field of play.
 
 You receive a stream of data that has been cleaned. Your task is to analyse it and output appropriate text to help the journalists.
 
 The data comprises:
 
 - shirt numbers (any `int`) -> text as per on field analysis
 - free form text (any `string`) -> the text unchanged
-- incidents in play (`Incident` enum) -> "RedCard", "Foul" etc.
+- incidents in play (any subclass of the type `Incident`) -> "Injury", "Foul" etc.
```

Comment:
As far as I can tell, we're not currently using the sub-classes anywhere in the example implementation. Maybe we should have separate cases for maybe one of the sub-classes and one case for the general base class?